src/test/java/org/takes/facets/auth/PsCookieTest.java

```diff
@@ -49,11 +59,42 @@ public void addsCookieToResponse() throws IOException {
             new RsPrint(
                 new PsCookie(
                     new CcPlain(), "foo", 1L
-                ).exit(new RsEmpty(), new Identity.Simple("urn:test:99"))
+                ).exit(new RsEmpty(), PsCookieTest.identity)
             ).print(),
             Matchers.containsString(
                 "Set-Cookie: foo=urn%3Atest%3A99;Path=/;HttpOnly;"
             )
         );
     }
+
+    /**
+     * The expiration date for a cookie must be a GMT time.
```

Comment:
@danff please check put this article about test method names and javadoc 
www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html?expand=1#test-method-names